### PR TITLE
Update the docs for `installing python-gammu`

### PR DESCRIPTION
**Pull request by Falke-Design**
I used the [README from python-gammu](https://github.com/gammu/python-gammu) as template to update the explanation in the docs.

I also changed `pip` to `pip3` but I'm not sure if this is correct.

[Demo](https://github.com/Falke-Design/gammu/blob/docs/python-gammu/docs/manual/project/install.rst)

### CODE REVIEW

File: docs/manual/project/install.rst [hunk 1]
```diff
@@ -562,9 +562,23 @@ To debug program crashes, you might want to build Gammu with
 with protections GCC makes and produce bogus back traces.
 
 
-Compiling python-gammu
+
+Installing python-gammu
 ----------------------
```
Comment (nijel) [inline docs/manual/project/install.rst:567]:
```suggestion
Installing python-gammu
-----------------------
```

**Comment by nijel** [comment]
Merged, thanks for your contribution!

**Comment by Falke-Design** [comment]
@nijel I currently updateing some parts in the docu, the most difficult part is to check if everything is rendered correct. First I used the Preview of Github but for that I always needed to commit. Now I installed `pandoc` and generate a html file over command line. What you using for testing the docs? Do you know a way to serve live rendered docs?

**Comment by nijel** [comment]
It's built using [Sphinx](https://www.sphinx-doc.org/).

I've now adjusted the CI to store built documentation in the artifact so that it can be downloaded: https://github.com/gammu/gammu/actions/runs/1991497708

**Pull request merged**